### DISCUSSION TITLE
how do i make this start from windows startup/bootup

### DISCUSSION THREAD

**Discussion by SuckyBuckyYT** [nicotine-plus/nicotine-plus#3192] [Q&A]
like i dont want the full gui to be opened just nicotine in the background openned and i can open full gui from the taskbar

**Comment by slook** [comment]
Copy a shortcut to the program into your Start Menu > Programs > 'Startup' folder,
and set Nicotine+ Preferences > User Interface > 'Minimize to tray on startup' -> On.
